### DISCUSSION TITLE
GWT4NB

### DISCUSSION THREAD

**Discussion by DaniJulia** [apache/netbeans#7536] [General]
### Apache NetBeans version

Apache NetBeans 22

### What happened

In https://netbeans.apache.org/tutorial/main/kb/docs/web/ajax-quickstart/ it states about a plugin repository in http://plugins.netbeans.org/ however, that webpage doesn't works.
How can I download the GWT4NB plugin?

### How to reproduce

http://plugins.netbeans.org/ doesn't works

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows, 11

### JDK

1.8

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by JoachimRohde** [comment]
Development of the GWT4NB plugin stopped five years ago. The last release / source can be found at https://github.com/ksfreitas/gwt4nb?tab=readme-ov-file
The last supported NetBeans version was 8.0

**Comment by DaniJulia** [comment]
I've also seen that link and it also does not work.
All the links to Downloads either the one from GitHub or the one from
plugins.netbeans.org, both are broken.
And from NetBeans IDE, there's no plugin available.
Thank you very much!

**Reply by Chris2011** [reply]
I already forked the repo years ago and I can have a look what is broken to make it work. But The features afaik were not that much. But will have a look soon.
